api/wb/serializers.py

```diff
@@ -4,18 +4,13 @@
 from rest_framework import exceptions
 
 from website.files import exceptions as file_exceptions
-from api.base.serializers import IDField, ShowIfVersion
+from api.base.serializers import IDField
 
 
 class DestinationSerializer(ser.Serializer):
     parent = ser.CharField(write_only=True)
     target = ser.CharField(write_only=True)
     name = ser.CharField(write_only=True, allow_blank=True, allow_null=True)
-    node = ShowIfVersion(
-        ser.CharField(write_only=True),
-        min_version='2.0', max_version='2.7',
-    )
-
```

Comment:
Note on the `node` field for reviewers.

I recommended removing versioning of obsolete node field on DestinationSerializer. This was for a private endpoint, that was written only to be called by WaterButler through a signed endpoint. It is not even in use yet, because we've had other priorities. I would assume no one else is using it either. Regardless, I don't think we have a responsibility to version these private endpoints.